pkgs/tools/networking/modem-manager/default.nix

```diff
@@ -30,7 +31,8 @@ stdenv.mkDerivation rec {
   '';
 
   postInstall = ''
-    mv $out/$out/etc/systemd/system/ModemManager.service $out/etc/systemd/system
+    # rename to modem-manager to be in style
+    mv $out/$out/etc/systemd/system/ModemManager.service $out/etc/systemd/system/modem-manager.service
```

Comment:
I think we stick to upstream here. Consistency is not worth breaking upstream documentation and expectation how things are named.

Reply:
Granted. I just found it weird renaming was done for `NetworkManager`, but not here. Should this also be reverted for `NetworkManager` then?

Reply:
Mhm. I would have not done it there too for the same reason.

Reply:
So it seems I have a different opinion then @domenkozar in 2014.

Reply:
I'd say as long as `NetworkManager` is renamed to `modem-manager`, `ModemManager` should be called `modem-manager`, too.

The current state is more than confusing.